webapp/src/components/modals/create_issue/create_issue.jsx

```diff
@@ -189,7 +189,7 @@ export default class CreateIssueModal extends PureComponent {
                     type='input'
                     required={true}
                     disabled={false}
-                    maxLength={65}
+                    maxLength={120}
```

Comment:
Have you tested the impact of this change? Just want to double check that the GH API accepts this.

Reply:
Hi @hanzei, thanks for the review. No issues found. Perhaps, a QA review can confirm:)